src/ring_theory/fractional_ideal.lean

```diff
@@ -235,22 +237,22 @@ include loc
 
 variables {P}
 
-lemma coe_to_fractional_ideal_injective (h : S ≤ non_zero_divisors R) :
-  function.injective (coe : ideal R → fractional_ideal S P) :=
-λ I J heq, have
-  ∀ (x : R), algebra_map R P x ∈ (I : fractional_ideal S P) ↔
-             algebra_map R P x ∈ (J : fractional_ideal S P) :=
-λ x, heq ▸ iff.rfl,
-ideal.ext (by simpa only [mem_coe_ideal, exists_prop, exists_mem_to_map_eq P h] using this)
+lemma coe_ideal_injective' (h : S ≤ non_zero_divisors R) :
+  function.injective (coe_ideal : ideal R → fractional_ideal S P) :=
+λ _ _ heq,
+le_antisymm ((coe_ideal_le_coe_ideal' S h).mp heq.le) ((coe_ideal_le_coe_ideal' S h).mp heq.ge)
```

Comment:
There are two assumptions labelled `h` here so I don't think it will work

Reply:
Well, I just want to cram it into one line, because the layout isn't very pretty currently (and I'd indent the second line). Now this version is exactly 100 chars: 
```
λ _ _ h', ((coe_ideal_le_coe_ideal' S h).mp h'.le).antisymm ((coe_ideal_le_coe_ideal' S h).mp h'.ge)
```